PPSTools/Utilities/interface/PPSOpticsCalibrator.h

```diff
@@ -0,0 +1,94 @@
+#ifndef PPS_OPTICS_CALIBRATION
+#define PPS_OPTICS_CALIBRATION
+#include "PPSTools/Utilities/interface/PPSUtilities.h"
+#include <boost/program_options.hpp>
+#include "TROOT.h"
+#include "Rtypes.h"
+#include "TH2F.h"
+#include "TTree.h"
+#include "TDirectory.h"
+#include "TSystem.h"
+#include "TRandom3.h"
+#include "TMath.h"
+#include "TLorentzVector.h"
+#include "TString.h"
+#include "TFile.h"
```

Comment:
essentially none of these includes are needed.. (and many will have quite a negative impact)

Reply:
TFile.h is needed in fact. The others has been removed.

Reply:
to clarify - 

when using only a pointer to a class in a header file, don't include the class's header, instead do 

class TFile;

when using a class in a cc file, include the corresponding header in the cc file itself.